scripts/test_permanent.ps1

```diff
@@ -1,56 +1,86 @@
+refreshenv
+
 $repo_root = [System.IO.Path]::GetDirectoryName((resolve-path "$PSScriptRoot"))
 
+$PATH_USER_BEFORE = [System.Environment]::GetEnvironmentVariable("PATH", "User")
+$PATH_MACHINE_BEFORE = [System.Environment]::GetEnvironmentVariable("PATH", "Machine")
+
+
+try {
+
 & "$repo_root/emsdk.ps1" install latest
-& "$repo_root/emsdk.ps1" activate latest --permanent
 
-$EMSDK_USER = [System.Environment]::GetEnvironmentVariable("EMSDK", "User")
-$EM_CONFIG_USER = [System.Environment]::GetEnvironmentVariable("EM_CONFIG", "User")
-$EMSDK_NODE_USER = [System.Environment]::GetEnvironmentVariable("EMSDK_NODE", "User")
-$EMSDK_PYTHON_USER = [System.Environment]::GetEnvironmentVariable("EMSDK_PYTHON", "User")
-$JAVA_HOME_USER = [System.Environment]::GetEnvironmentVariable("JAVA_HOME", "User")
-$EM_CACHE_USER = [System.Environment]::GetEnvironmentVariable("EM_CACHE", "User")
-$PATH_USER = [System.Environment]::GetEnvironmentVariable("PATH", "User")
+$esc = '--%'
+& "$repo_root/emsdk.ps1" activate latest $esc $env:PERMANENT_FLAG $env:SYSTEM_FLAG
+
+if ($env:SYSTEM_FLAG) {
+    $path_type="Machine"
```

Comment:
Maybe `path_type` is the wrong name since it applies to all variable?  How about `env_type`?  Or `reg_type`?

Reply:
Fixed in https://github.com/emscripten-core/emsdk/pull/642/commits/3b00b38c358204f059cf128d6a332d610ceaed54